src/components/TeamForm.tsx

```diff
@@ -0,0 +1,76 @@
+const TeamForm = () => {
+  return (
+    <div className="min-h-screen flex items-center justify-center bg-white border border-gray-600 p-6">
+      <form className="w-full max-w-md bg-black bg-opacity-50 p-8 bg-gradient-to-br from-gray-900 to-purple-700 rounded-lg">
+        <h2 className="text-2xl font-semibold text-white mb-6">
+          Tell us about your team
+        </h2>
+        <label htmlFor="companyName" className="block text-gray-300 mb-2">
```

Comment:
Ideally the input should focu on clicking the label text. But it's not happening